pkg/providers/vsphere/virtualmachine/backup.go

```diff
@@ -271,12 +272,26 @@ func getDesiredVMResourceYAMLForBackup(
 		return "", err
 	}
 
+	// Use a copy to avoid modifying the VM object at the end of reconciliation.
+	copyVM := vm.DeepCopy()
```

Comment:
Line [168](https://github.com/vmware-tanzu/vm-operator/pull/698/files#diff-5b4870681742b58d08e67601c0496f92bade48b9cb23fe808895e318abb13ccfR169) in this file is also creating a deep copy of the VM.  Can we please dedupe that?

Reply:
They aren’t duplicated because that line will only be executed when the incremental FSS is enabled, and this function is called when the FSS is disabled. I thought about getting a deep copy before the FSS check, but it's not always needed when the FSS is enabled (e.g. when any of the `!ok || backupVersionAnnotation == "" || len(ecToUpdate) != 0 || !vmBackupInSync` returns false at line 161).